lib/kubernetes/client.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+class Kubernetes::Client
+  SERVICE_FINALIZER = "k8s.ubicloud.com/load-balancer"
+
+  def initialize(kubernetes_cluster, session)
+    @session = session
+    @kubernetes_cluster = kubernetes_cluster
```

Comment:
you can also add @lb = LoadBalancer.... here to avoid querying for the LB in the methods